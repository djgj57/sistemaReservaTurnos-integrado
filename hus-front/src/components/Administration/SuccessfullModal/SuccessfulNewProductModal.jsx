import React from "react";
import {
  Overlay,
  ModalContainer,
  TextContainer,
} from "./SuccessfulNewProductModalStyled";
import Check from "./../../../images/Check";
import Button from "./../../shared/Button/Button";
import Lottie from "react-lottie";

const SuccessfulNewProductModal = ({ stateModal, setStateModal }) => {
  const defaultOptions = {
    loop: false,
    autoplay: true,
    animationData: require("./../../../assets/tick_success.json"),
    rendererSettings: {
      preserveAspectRatio: "xMidYMid slice",
    },
  };
  return (
    <>
      {stateModal && (
        <Overlay>
          <ModalContainer>
            <Lottie options={defaultOptions} height={150} width={150} />
            <TextContainer>
              <p>Tu propiedad se ha creado con Ã©xito.</p>
            </TextContainer>
            <Button
              handlerEvent={() => setStateModal(false)}
              to="/"
              // theme="secondary"
              text="Volver"
            />
          </ModalContainer>
        </Overlay>
      )}
    </>
  );
};

export default SuccessfulNewProductModal;
