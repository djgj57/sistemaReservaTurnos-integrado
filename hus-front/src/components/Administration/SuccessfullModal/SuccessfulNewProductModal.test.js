import React from "react";
import "@testing-library/jest-dom/extend-expect";
import { cleanup, render, screen } from "@testing-library/react";
import SuccessfulNewProductModal from "./SuccessfulNewProductModal.jsx";
import { MemoryRouter } from "react-router";

test("SuccessfulNewProductModal renders correctly", () => {
  const { container } = render(
    <MemoryRouter>
      <SuccessfulNewProductModal />
    </MemoryRouter>
  );
  expect(container).toMatchSnapshot();
});
