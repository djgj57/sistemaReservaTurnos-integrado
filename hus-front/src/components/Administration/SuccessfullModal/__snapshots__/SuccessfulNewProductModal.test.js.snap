// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SuccessfulNewProductModal renders correctly 1`] = `<div />`;
